Formula/ntopng.rb

```diff
@@ -32,6 +32,7 @@ class Ntopng < Formula
   depends_on "automake" => :build
   depends_on "gnutls" => :build
   depends_on "json-glib" => :build
+  depends_on "libsodium" => :build
```

Comment:
I added direct dependency to ntopng because ntopng does [directly reference libsodium](https://github.com/ntop/ntopng/blob/5e649a2d1130b4a3ab0c5bb673d615172cc0bdbb/Makefile.in#L77-L84) in its build files and I thought this might help ensure appropriate rebuilds are done.

Reply:
That should be a general dependency, not a build-only one, since you'll be linking `ntopng` against `libsodium` shared libraries, which must be available at run-time.

Reply:
Fixed, thanks! The other dependencies made me suspect it was going to be statically linked, like zeromq itself, but I guess not.